org.jacoco.core.test.java7/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2009, 2018 Mountainminds GmbH & Co. KG and Contributors
+   All rights reserved. This program and the accompanying materials
+   are made available under the terms of the Eclipse Public License v1.0
+   which accompanies this distribution, and is available at
+   http://www.eclipse.org/legal/epl-v10.html
+
+   Contributors:
+      Evgeny Mandrikov - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jacoco</groupId>
+    <artifactId>org.jacoco.tests</artifactId>
+    <version>0.8.2-SNAPSHOT</version>
+    <relativePath>../org.jacoco.tests</relativePath>
+  </parent>
+
+  <artifactId>org.jacoco.core.test.java7</artifactId>
+
+  <!--<name></name>-->
+
+  <properties>
+    <jacoco.includes>org.jacoco.core.*</jacoco.includes>
```

Comment:
I would prefer to not record coverage for integration tests at all. In this case this property is not relevant.

Reply:
They are not included into final report.

All classes are included into instrumentation and agent is enabled by default - see https://github.com/jacoco/jacoco/blob/753d6532d56892d037ff7958ea30c9bc27f6e8e4/org.jacoco.tests/pom.xml#L49-L70

Thus if we want to exclude all classes from instrumentation, then this should be replaced by `jacoco.excludes`? Or if we want to completely disable agent, then `jacoco.skip`?

Reply:
Yep, I think JaCoCo agent should not be in the game for integration tests. Targets are instrumented programmatically anyways during the test. So `jacoco.skip` would be the best fit here (and `jacoco.includes` can be removed at all).

Reply:
Replaced `jacoco.includes` by `jacoco.skip`.